nekoyume/Assets/_Scripts/UI/Widget/WorldMap.cs

```diff
@@ -258,7 +258,25 @@ private void ShowWorld(int worldId)
         {
             if (!SharedViewModel.WorldInformation.TryGetWorld(worldId, out var world))
             {
-                throw new ArgumentException(nameof(worldId));
+                var unlockConditionRow =
+                    TableSheets.Instance.WorldUnlockSheet.OrderedList
+                        .FirstOrDefault(row =>
+                            row.WorldIdToUnlock == worldId);
+                if (unlockConditionRow is null)
+                {
+                    throw new ArgumentException(nameof(worldId));
+                }
+
+                if (!SharedViewModel.WorldInformation
+                        .IsStageCleared(unlockConditionRow
+                        .StageId))
+                {
+                    throw new Action.InvalidWorldException();
```

Comment:
The `Action.InvalidWorldException` is a bit awkward.

Reply:
🤔 I agree. Exception of action doesn't seem to fit in with this situation.
Is there a better way?